src/main/java/org/jenkinsci/remoting/engine/EngineUtil.java

```diff
@@ -58,6 +58,23 @@ public static String readLine(InputStream inputStream) throws IOException {
     }
 
     /**
+     * Read a certain amount of characters from the stream.
+     *
+     * @param inputStream The input stream to read from.
+     * @param len The amount of characters to read.
+     * @return The characters read.
+     * @throws IOException
+     */
+    public static String readChars(InputStream inputStream, int len) throws IOException {
```

Comment:
It was a method for tests only. Makes sense to use package-internal wrapper next time